src/vs/workbench/browser/parts/titlebar/titlebarPart.ts

```diff
@@ -552,7 +530,7 @@ export class BrowserTitlebarPart extends Part implements ITitlebarPart {
 		// --- Editor Actions
 		const activeEditorPane = this.editorGroupsContainer.activeGroup?.activeEditorPane;
 		if (activeEditorPane && activeEditorPane instanceof EditorPane) {
-			const result = activeEditorPane.getActionViewItem(action);
+			const result = activeEditorPane.getActionViewItem(action, { hoverDelegate: this.hoverDelegate });
```

Comment:
I had somehow expected the `options` being passed into the `actionViewItemProvider` method, didn't you make it required? Or only in other places.